packages/api/src/routes/medical/patient.ts

```diff
@@ -310,6 +314,46 @@ router.post(
   })
 );
 
+/**
+ * GET /patient/:id/medical-record
+ *
+ * Returns the url to download a patient's medical record summary, if it exists.
+ *
+ * @param req.cxId The customer ID.
+ * @param req.param.id The ID of the patient whose data is to be returned.
+ * @param req.query.conversionType Indicates how the medical record summary should be rendered. Accepts "pdf" or "html".
+ */
+router.get(
+  "/:id/medical-record",
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getCxIdOrFail(req);
+    const patientId = getFrom("params").orFail("id", req);
+    const type = getFrom("query").orFail("conversionType", req);
+    const conversionType = consolidationConversionTypeSchema.parse(type);
+
+    const url = await getMedicalRecordSummary({ patientId, cxId, conversionType });
+    return res.json({ url });
+  })
+);
+
+/**
+ * GET /patient/:id/medical-record-status
+ *
+ * Checks if a patient's medical record summary exists in either PDF or HTML format and the date it was created.
+ *
+ * @param req.cxId The customer ID.
+ * @param req.param.id The ID of the patient whose data is to be returned.
+ */
+router.get(
+  "/:id/medical-record-status",
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getCxIdOrFail(req);
+    const patientId = getFrom("params").orFail("id", req);
+    const status = await getMedicalRecordSummaryStatus({ patientId, cxId });
+    return res.json(status);
```

Comment:
I suggest we go back to use DTOs for API responses. Returning the type from the cmd limits our ability to change the cmd in the future.

Also, it makes the cmd defining its return type, and not importing it from the SDK 😢